docs/ff-concepts/localization-accessibility/localization.md

```diff
@@ -0,0 +1,221 @@
+
+These options are accessible through **Set from Variable > Internationalization**.
+
+![retrieve-lang-data.png](imgs/retrieve-lang-data.png)
+
+## Formatting Dates, Numbers, and Currency
+
+Localization isn't just about translating strings. It also involves adapting dates, numbers, and currencies to the user's locale.
+
+### Dates
+
+To ensure your app displays dates in formats familiar to users from different regions, you can use the predefined **DateTime Format Options** while displaying dates.
+
+For example, in the United States, dates follow a **month, day, year** format (e.g., 12/31/2023), whereas in India, they use a **day, month, year** format (e.g., 31/12/2023). To accommodate these regional differences, set the format option to `yMd`, a locale-aware format that automatically adjusts date representation based on the user's locale.
+
+![localize-dates.avif](imgs/localize-dates.avif)
+
+:::tip
+
+Here are a few more locale-aware formatting options you can use:
+
+- **`yMMMd`** – Formats the date with an abbreviated month and day, e.g., `Dec 31, 2023` (US) or `31 Dec 2023` (India).
+- **`jm`** – Displays time with minutes, e.g., `5:30 PM` (US) or `17:30` (Europe).
+
+For custom locale-specific date formats, you can also [**create your own patterns**](../../resources/data-representation/global-properties.md#custom-formatting).
+
+:::
+
+### Numbers
+
+Different regions use different symbols for decimal and thousand separators. For example, the U.S. uses a period for decimals and a comma for thousands, while many European countries use the opposite.
+
+To localize the numbers, set the [**Number Format Options**](../../resources/ui/widgets/basic-widgets/text.md#formatting-numbers) to **Decimal** and then set the **Decimal Type** to **Automatic**.
+
+![localize-numbers.avif](imgs/localize-numbers.avif)
+
+### Currency
+
+Currency symbols and their placement vary by locale. For example, in the U.S., the dollar sign appears before the amount (`$1,000.00`), whereas in countries like France, the currency symbol is placed after the amount (e.g., `1 000,50 €`).
+
+To handle this behavior, enable the **Display as Currency** option under Number Format settings and leave the **Currency Symbol** field empty to automatically adjust based on the user’s locale.
+
+![localize-currency.avif](imgs/localize-currency.avif)
+
+## Testing
+
+Localization testing is crucial to ensure that all elements work properly across different languages and locales. Here are a few ways to test localization:
+
+- **Change Device Locale**: Test your app by changing the device locale to verify translations and layout adjustments.
+- **Use Emulators**: Use Android or iOS emulators to simulate different locales and ensure everything is displaying correctly.
+- **Long Texts**: Verify that long translations do not overflow or cause UI issues.
+- **Manual Testing**: Manually verify the accuracy of translations, date formats, number formats, etc.
```

Comment:
I think somewhere in the doc (either here or as an example) we should mention about handling RTL languages

Reply:
I looked into it, but we don't support RTL languages natively in FF.

Reply:
Ooh interesting!